packages/dashboard/src/_stories/_mock/dashboard-filter-build.json

```diff
@@ -0,0 +1,2839 @@
+{
```

Comment:
@Matt-DiPaolo actually can we delete out all of the data that isn't being used? should be able to do it with a regex.